docs/development/docs.rst

```diff
@@ -10,27 +10,50 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation, you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+1. click the checkmark next to your commit and it will expand to have multiple options
+2. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+3. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+If you're making large changes to the documentation, you'll want to verify those changes locally before pushing upstream.
+
+1. follow the instructions to `Set up your environment <https://docs.readthedocs.io/en/stable/development/install.html#set-up-your-environment>`__
```

Comment:
I don't think we actually want users to _install Read the Docs_ to contribute docs. People should be able to just have a Sphinx environment setup locally and build the docs through that. 

As you've seen in this PR, having a full RTD install is _a ton of work_ and requires understanding a great depth of technical stuff. I think we should strongly suggest folks just do work in git and let the PR builds do the building, and alternatively to just get a local Sphinx install.

Reply:
Ah, I see what you're saying. Agreed, that would be much easier. I will go through this path myself then update the instructions here.

Reply:
@cocobennett Sounds good. Having a local install is great (and you've already found lots of bugs :D) but keeping this barrier to entry low is definitely going to be a benefit. Thanks again for your help on this!